core/pom.xml

```diff
@@ -209,6 +215,15 @@
                </execution>
             </executions>
          </plugin>
+         <plugin>
+            <groupId>org.apache.karaf.tooling</groupId>
+            <artifactId>karaf-maven-plugin</artifactId>
```

Comment:
Please move this before the defaults plugin, since the bundle plugin, the helper and the karaf plugin all seem related.